library/init/meta/interactive.lean

```diff
@@ -6,8 +6,11 @@ Authors: Leonardo de Moura
 prelude
 import init.meta.tactic init.meta.rewrite_tactic init.meta.simp_tactic
 import init.meta.smt.congruence_closure init.category.combinators
+import init.meta.lean_parser
 
 namespace interactive
+@[reducible] meta def parse {α : Type} (p : lean_parser α) : Type := α
```

Comment:
> We can keep itactic as is. I think this would cover everything we have right now.

Yeah, I think we got it. Phew :) . We can always add a weakly-typed alternative to `parse` in the future, but for now I would go with the shorter name since it is _everywhere_ in the signatures.

Unfortunately, I don't think I will be able to create a working implementation before starting my trip tomorrow.